lib/msal-common/src/request/RequestValidator.ts

```diff
@@ -7,6 +7,7 @@ import { StringUtils } from "../utils/StringUtils";
 import { ClientConfigurationError } from "../error/ClientConfigurationError";
 import { PromptValue, CodeChallengeMethodValues} from "../utils/Constants";
 import { StringDict } from "../utils/MsalTypes";
+import { omitBy, isEmpty } from "lodash";
```

Comment:
Prefer not to introduce a new dependency if it can be avoided. I think what you're using this for can be accomplished without this lib

Reply:
Ah, I see now that the existing lodash package is located in devDepencies instead of dependencies.